components/modal/confirm.tsx

```diff
@@ -28,6 +29,7 @@ const ConfirmDialog = (props: ConfirmDialogProps) => {
   const okText = props.okText ||
     (okCancel ? runtimeLocale.okText : runtimeLocale.justOkText);
   const cancelText = props.cancelText || runtimeLocale.cancelText;
+  const autoFocusButton = props.autoFocusButton === null ? false : props.autoFocusButton || 'ok'
```

Comment:
why do we need this line? We can simply use props.autoFocusButton directly. 
```
const { autoFocusButton } = this.props;
```

Reply:
Wouldn't that give 'ok' when null is passed? Passing null serves to disable autofocus in this case.